BlankUtility/src/Logging/Logger.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+//shared_ptr
+#include <memory>
+
+#include "../Macros.h"
+
+#include "spdlog/spdlog.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
+#include "spdlog/sinks/basic_file_sink.h"
+
+// Logger Macros
+#define BLANK_ENGINE_TRACE(...) Blank::Logger::getLogger()->trace(__VA_ARGS__) // VA ARGS just means forwards arguments from (...)
```

Comment:
What do you think of renaming the macros to something like LOG_TRACE

Reply:
Sounds good, will get on it in a second.